### PR TITLE
New component: Widget

### PR DESCRIPTION
**Pull request by TaraldRotsaert**
### Description
- This PR introduces a new Widget component to the ahoy library. This component consists of a body and has the possibility to get a header and/or footer.

#### Screenshots
Screen Shot 2019-04-01 at 14 54 25
Screen Shot 2019-04-01 at 14 54 31
Screen Shot 2019-04-01 at 14 54 47

### Breaking changes
- None

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ addDecorator(
   withBackgrounds([
     { name: 'White', value: '#ffffff', default: true },
     { name: 'Aqua lightest', value: '#e6f2ff' },
+    { name: 'Neutral', value: '#F7F7FA' },
```
Comment (driesd) [inline .storybook/config.js]: To be consistent, maybe you can set the color hex value in lowercase?

File: src/components/widget/Widget.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        })}
+      </Box>
+    );
+  }
+}
+
+Widget.propTypes = {
+  /** The content to display inside the widget. */
+  children: PropTypes.node,
+  size: PropTypes.oneOf(Object.keys(SIZES)),
```
Comment (driesd) [inline src/components/widget/Widget.js]: Can you add the prop type description for `size`?

File: src/components/widget/Widget.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  medium: 4,
+  large: 5,
+};
+
+class Widget extends PureComponent {
+  render() {
+    const { children, size, ...others } = this.props;
+
+    return (
+      <Box className={theme['widget']} {...others}>
```
Comment (driesd) [inline src/components/widget/Widget.js]: We should also provide the possibility to add an additional class name (with `cx`).

**Comment by TaraldRotsaert** [comment]
The segmented island could definitely be something to look into and i am aware of the datagrid but i am going to introduce some changes in a different pr for that one.

**Pull request merged**